hack/local-up-cluster.sh

```diff
@@ -646,8 +647,14 @@ function start_cloud_controller_manager {
       node_cidr_args="--allocate-node-cidrs=true --cluster-cidr=10.1.0.0/16 "
     fi
 
+    if [[ -z ${EXTERNAL_CLOUD_PROVIDER_EXE} ]]; then
```

Comment:
EXE is so DOS'ish. Don't we call that "binary" or "command"?

Reply:
:) ack switching to BINARY